### DISCUSSION TITLE
[archived] "Other Use Cases" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#11673] [General]
# argo-workflows/use-cases/other/

https://argoproj.github.io/argo-workflows/use-cases/other/?utm_source=argo-ui

**Comment by carlosjgp** [comment]
I just need the Argo Events UI

**Reply by agilgur5** [reply]
Not sure what you mean. [Argo Events](https://github.com/argoproj/argo-events) is a different project from Argo Workflows

**Reply by carlosjgp** [reply]
yeah I found it very strange that the Argo Events UI is only available on Argo Workflows and not as an independent thing or plugin

**Reply by agilgur5** [reply]
Oh. I didn't even know it didn't have its own separate UI. TIL
